### PR TITLE
WooFound Image Titles

### PR DESCRIPTION
**Pull request by skcain**
Make each image more clear

### CODE REVIEW

**Comment by pacarole** [comment]
🚢

File: woofound.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <a class="basic-button" href="assessment.html">Back</a>
   </div>
   <div class="centered">
+    <p class="wooFoundImageTitle">Working with computers</p>
```
Comment (antislice) [inline woofound.html]: Maybe something more vague like "Using technology"?

**Comment by antislice** [comment]
I had a tiny comment about wording, also maybe we could visually associate the title with the picture more BUT neither of these are something that needs to happen before your interview.
🚢 🚢 🚢

**Comment by skcain** [comment]
Changed computers to technology. Although I wonder if the term "technology" may be too jargon-y for some people. As far as the visual association, I figure we can work on that more with later iterations.

**Pull request merged**